### ISSUE TITLE
New contributor outreach

### ISSUE THREAD

**Issue by JamieDanielson** [open-telemetry/sig-contributor-experience#17]
Reaching out to new contributors and getting feedback to understand what is currently difficult.

Answer questions they may have to help unblock faster, encourage them to update documentation.

**Comment by svrnm** [comment]
I like this very much!

**Comment by theletterf** [comment]
Great one! Joining the initiative. I wonder if there'd be some way of automating this following CLA's signature and PR merge.

**Comment by svrnm** [comment]
> Great one! Joining the initiative. I wonder if there'd be some way of automating this following CLA's signature and PR merge.

The GitHub API allows you to check the `author_association` and if someone is doing their first issue/PR it will tell you `FIRST_TIME_CONTRIBUTOR`, I am not sure if we can reverse search for those or just need to look at the stream of all incoming issues/PRs

**Comment by mx-psi** [comment]
I self-assigned myself to this :)

**Comment by JamieDanielson** [comment]
I'm also curious if we want to do anything with folks who attend SIG meetings - there are typically a group of "regulars" every week but sometimes a new face shows up. It could be useful to understand why they attended the meeting, how they found out about it,i are they getting the information they need, etc.

**Comment by theletterf** [comment]
In general, I believe it should be easier to find and join SIG meetings. It's not hugely evident where to find the shared calendar, Zoom links, etc. (not sure if this is by design). Thus, it's also harder to get bystanders to join in.

**Comment by svrnm** [comment]
Copying @mx-psi and my brainstorming notes from yesterdays meeting:

Brainstorm on what “new contributor outreach means”? What is in scope? What is it we want to do?
* Spaces for new contributors (Slack channel, Monthly Zoom call, GH Discussions…)
* Mentorship program
* https://up-for-grabs.net/#/filters?tags=opentelemetry
  * Filters on “help wanted” or “good first issue”
  * There are some projects who give a lot of meaning to those labels, like the issue is well scoped, helps the project, easy for beginners BUT also they commit resources to help newcomers that are going to work on it.
* Interview/Feedback for first time contributors (could be a form, but also a 1:1 slack conversation)
* Way to identify type of contributors we have (paid/non-paid, one-time-solve-my-own-problem or interested in continuous contribution)
  * They have different needs, we need to address differently
    * Paid contributors are “here to stay”, but still might run into headwinds, so we should help them to have a good and rapid onboarding
    * One time sole my own problem contributors just need their thing to be fixed
    * “Volunteer”, Casual/Non-Paid contributors … (maybe related to https://github.com/open-telemetry/sig-contributor-experience/issues/20?) 
      * Potentially a component owner? Give them their thing to do + the right amount of recognition (they need to be seen otherwise they might go away once again)?
        * Tiny things like including them in the release notes, etc.
        * What else?
    * This is of course not a fixed set of groups, people are moving from one to the other or there are some boundaries (someone might be paid to work ~5hrs per week on the project, vs a volunteer who spends ~10hrs into the project)
* On the membership issue, ask more about the people? What type of contributor they are? What motivates them? 
How many members does the Org have and how many of them are “active” contributors (have contributed in the last ~3months)

**Comment by svrnm** [comment]
> I'm also curious if we want to do anything with folks who attend SIG meetings - there are typically a group of "regulars" every week but sometimes a new face shows up. It could be useful to understand why they attended the meeting, how they found out about it,i are they getting the information they need, etc.

Agreed, and to be honest I am also always not 100% sure how to address them. Some people just show up to listen in, some might not even want to be approached. Maybe sending a message via the meeting chat would be a low barrier of entry?

> In general, I believe it should be easier to find and join SIG meetings. It's not hugely evident where to find the shared calendar, Zoom links, etc. (not sure if this is by design). Thus, it's also harder to get bystanders to join in.

This is not by design, just that nobody improved that! We could probably have some of that on the website as well. Maybe rename the "Community" to "Contributors" or have a CTA on the landing page around "Contribute", etc.

**Comment by mx-psi** [comment]
A possible strategy for attracting volunteer contributors that I discussed with the Docs SIG before is to show in the blog the interesting technical challenges the SIGs face on their day to day. This can make contributing to OTel feel more prestigious or appealing and is also good just for getting the project to be known.
